### PR TITLE
Add links to some specific flowdock flows

### PR DESCRIPTION
**Pull request by wdoug**
#### What does this PR do?

As titled

### CODE REVIEW

**Comment by evanplaice** [comment]
Nice. It would better, UX-wise, to see the content be broken up into separate pages.

For Example:

### Our Mission

Less focus on 'who we are' more focus on 'what we accomplish'. With showcases of the people responsible for doing the work.

- Learning opportunities for members
- Community outreach
- etc

IMO, CodeForDenver.org already covers this.

### Our Process

Process involves more than tools. IMO, most of it is people-oriented. What can new-comers expect. What insight can the site give to people, not just those directly involved with development.

In short, online onboarding.

### Get Involved

Want to join the team. Here's how. Break this up into sections that are more skill-oriented.

- Designers
- Developers
- Artists
- Organizers
- Promoters

### Make a Difference

What's the impact of the work being done?

- Projects completed
- Gallery of outreach events
- Kudos for active members

Anyway, in the mean time it's good to keep adding content.

**Review by evanplaice — COMMENTED** [review]
LGTM

**Review by dviramontes — APPROVED** [review]
👍

**Comment by wdoug** [comment]
@evanplaice these are great suggestions! Would you mind either creating a pull request, or at least creating a waffle card (or multiple) to add these suggestions?
